### DISCUSSION TITLE
Script downloader/Updater

### DISCUSSION THREAD

**Discussion by kotarokun** [007revad/todo_list#17] [Ideas]
Hello

i love your scripts. 
i have sometime enough time for everything, and sometimes not. it is possible to create an script that will just download an Script which I chose, or maybe update my scripts with the new one especialy the download-Scirpt itself for new repos...

greets

**Comment by 007revad** [comment]
Most of my scripts can update themselves.
1. The scripts that can be scheduled have an autoupdate option. If the autoupdate option is not used it will ask if you want to update.
2. The scripts that can't be scheduled will ask if you want to update.

So you want a script that shows a list of my scripts, including new ones, so you can select one to download?

For it to be able to also show which scripts have newer versions available the script would need to know which scripts you have installed. I could have each script update a database with the script's location and version.
